src/algorand/AlgorandTools.ts

```diff
@@ -91,14 +91,14 @@ export class AlgorandTools implements EdgeCurrencyTools {
     const { pluginId } = this.currencyInfo
     const networks = { [pluginId]: true }
 
-    const { parsedUri, edgeParsedUri } = parseUriCommon(
-      this.currencyInfo,
-      uri,
-      networks,
-      this.builtinTokens,
-      currencyCode ?? this.currencyInfo.currencyCode,
-      customTokens
-    )
+    const { parsedUri, edgeParsedUri } = parseUriCommon({
+      currencyInfo: this.currencyInfo,
+      uri: uri,
```

Comment:
For this whole commit, do a search & replace for all `uri: uri,` to just `uri,`, and similarly for `network` and `customTokens`.